operations/firmware/FAS_Use_Cases.md

```diff
@@ -388,7 +388,12 @@ The CMM firmware update process also checks and updates the Cabinet Environmenta
 
     The `--prereq` option ensures all required components are powered on first. The `--continue` option allows the command to complete in systems without fully populated hardware.
 
-6.  After the components have powered on, boot the nodes using the Boot Orchestration Services \(BOS\).
+6.  Bring up the Slingshot Fabric
+    To bring up the Slingshot fabric, see:
+    -  The *Slingshot Administration Guide* for Cray EX systems PDF.
```

Comment:
Change to:
 
The *Slingshot Administration Guide* PDF for HPE Cray EX systems.